README.md

```diff
@@ -224,7 +225,7 @@ docker-compose.yml
           - "9950:9950"
           - "9952:9952"
         volumes:
-          - ./settings/settings.json:/usr/src/app/settings.json
+          - ./nano_proxy/settings.json:/usr/src/app/settings.json
```

Comment:
Please also change the description above from "..to a new folder settings/settings.json" to "..to a new folder nano_proxy/settings.json"

And from "(at the root of the settings folder)" to "(at the root of the nano_proxy folder)"

Reply:
Didn't see those, thanks for pointing out!